api/client/client.go

```diff
@@ -1280,3 +1280,28 @@ func (c *Client) DeleteAllNodes(ctx context.Context, namespace string) error {
 	_, err := c.grpc.DeleteAllNodes(ctx, &types.ResourcesInNamespaceRequest{Namespace: namespace}, c.callOpts...)
 	return trail.FromGRPC(err)
 }
+
+// GetClusterNetworkingConfig gets cluster networking configuration.
+func (c *Client) GetClusterNetworkingConfig(ctx context.Context, skipValidation bool) (types.ClusterNetworkingConfig, error) {
```

Comment:
do we need to expose `skipValidation` at the API level?
AFAIK it's only relevant when the auth server reads data from the backend